.github/workflows/checks-integration.yml

```diff
@@ -0,0 +1,50 @@
+name: Checks (Integration)
+
+on:
+  push:
+    branches:
+      - master
+  schedule:
+    - cron: '0 23 * * SUN-THU'
+  workflow_dispatch:
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref == 'refs/heads/master' && github.run_number || github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  checks:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
```

Comment:
The scheduled run might not be intended to run in forks. But, we can work on it in a follow-up PR.

https://github.com/optuna/optuna/blob/fc5e262042a60bbcc888541ce7fa5e08769ac478/.github/workflows/speed-benchmarks.yml#L11-L12